src/hosted-buttons/utils.js

```diff
@@ -30,7 +35,7 @@ const getHeaders = (accessToken?: string) => ({
 });
 
 export const getMerchantID = (): string | void => {
-  // The SDK supports mutiple merchant IDs, but hosted buttons only
+  // The SDK supports multiple merchant IDs, but hosted buttons only
   // have one merchant id as a query parameter to the SDK script.
   // https://github.com/paypal/paypal-sdk-client/blob/c58e35f8f7adbab76523eb25b9c10543449d2d29/src/script.js#L144
   const merchantIds = getSDKMerchantID();
```

Comment:
we should update this and the error message, this is a feature called Multi-Seller Payments (MSP) https://developer.paypal.com/docs/multiparty/checkout/multiseller-payments/

Reply:
I think this is actually a limitation of the backend - it only accepts one merchant ID in the request payload

Reply:
oh yeah, I agree, I mean we should update our comment and our error message to call out that the MSP feature is unsupported

Reply:
sounds good, I've updated the comment block here